data/ecosystems/h/hedera.toml

```diff
@@ -8,6 +8,7 @@ sub_ecosystems = [
   "Buidler Labs",
   "Burstall",
   "Calaxy",
+  "CoreLabs",
```

Comment:
CoreLabs ecosystem doesn't exist in the taxonomy so you'll need to create a .toml file accordingly. You can check the README for further details or even other merged PRs that you can use as reference, if you need examples.